frontend/src/pages/Auth/SignUp.tsx

```diff
@@ -87,6 +87,11 @@ export const SignUp: React.FC = () => {
 					provider: additionalUserInfo.providerId,
 				})
 
+				analytics.trackGaEvent('sign_up', {
+					email: user.email!,
+					provider: additionalUserInfo.providerId,
```

Comment:
any other user attributes we couldsend in this obj? maybe their name? workspace id if it's available at this moment?

Reply:
I don't think we have any of that yet because all they've submitted is their email and password 🤔